tap_hubspot/__init__.py

```diff
@@ -43,7 +43,9 @@
     "emails": {"bookmark_key": "lastUpdated"},
     "campaigns": {},
     "communications": {"bookmark_key": "updatedAt"},
-    "communication_properties": {"bookmark_key": "updatedAt"}
+    "communication_properties": {"bookmark_key": "updatedAt"},
+    "engagements__c": {"bookmark_key": "updatedAt"},
```

Comment:
this will try to retrieve data for all hubspot user, I suggest you write the logic to only sync for a specific portal id

Reply:
right now engagements__c will be the table name in bigquery, but I suggest the table name look like this in bigquery `f"p{portalId}_marketing_engagements"`. Let me know if you have another idea